test/test_helper.rb

```diff
@@ -27,3 +27,9 @@
 
   # Add more helper methods to be used by all tests here...
 end
+
+if defined?(FactoryBot)
```

Comment:
I don't think you need the conditional anymore since FactoryBot is a dependency in the test group.